packages/thumbprint-react/components/ModalDefault/index.module.scss

```diff
@@ -204,6 +204,13 @@ $width-wide: 1400px;
     padding-top: 12px;
 }
 
+// The double selectors in this file ensure that consumers using the `className` prop can't easily
+// overwrite the styles that we want to enforce in the component.
+.modalContentFullBleed.modalContentFullBleed {
+    margin: 0 -32px !important;
+    padding: 0 32px !important;
```

Comment:
Let's use a variable that is shared with `ModalContents` so that this is more likely to stay in sync in the future.

Reply:
done